src/Illuminate/Foundation/Exceptions/Handler.php

```diff
@@ -391,16 +391,14 @@ protected function whoopsHandler()
      */
     protected function renderHttpException(HttpException $e)
     {
-        $status = $e->getStatusCode();
-
         $paths = collect(config('view.paths'));
 
         view()->replaceNamespace('errors', $paths->map(function ($path) {
             return "{$path}/errors";
         })->push(__DIR__.'/views')->all());
 
         if (view()->exists($view = "errors::{$status}")) {
```

Comment:
Did you forget here? It being used twice, so having a temporary variable does make sense.

Reply:
Thank you very much! I am negligent